### ISSUE TITLE
Update just one repo

### ISSUE THREAD

**Issue by kenkouot** [Masterminds/glide#474]
There seems to be conflicting information from #92 and the official Glide documentation/cli help. Is there currently anyway to update just one repository using Glide?

Running `glide update github.com/foo/bar` doesn't seem to update that one repository.

I've been manually replacing hashes in my lock file and running `glide install` to get the desired outcome. Would love to know a better way, if there is one.

**Comment by sdboyer** [comment]
Yeah, there's a lot of discussion around this. Probably the best issue for the full view of what's upcoming is #252 . tl;dr - it's very much a planned feature, but one that will likely only get here once the new engine is in place.

till then...well, yeah, I do that same, awkward hash editing thing that you do 😦

**Comment by atombender** [comment]
This should closed in favour of #252.

**Comment by technosophos** [comment]
Closed as a duplicate of #252
